fh-mbaas-template-3node.json

```diff
@@ -1677,6 +1677,12 @@
       "value": "info"
     },
     {
+      "name": "MONGODB_PVC_SIZE",
+      "description": "The size of the volume for MongoDB Data",
+      "value": "50Gi",
+      "required": true
+    },
+    {
       "name": "SMTP_SERVER",
```

Comment:
Yeah, I'm not actually saying we should remove it as I know it will default to the 50Gi. Just opening a discussion really. If we param it we need to doc it also I guess.